### ISSUE TITLE
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 415974: invalid continuation byte

### ISSUE THREAD

**Issue by ClimberMel** [isaacharrisholt/quiffen#51]
I am trying out your quiffen module and I am getting errors when trying to read in a QIF file.
Here are the errors:
```
site-packages\quiffen\core\qif.py", line 133, in parse

data = path.read_text(encoding="utf-8").strip().strip("\n")
  File "\Python310\lib\pathlib.py", line 1135, in read_text
    return f.read()
  File "\Python310\lib\codecs.py", line 322, in decode
    (result, consumed) = self._buffer_decode(data, self.errors, final)
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xe9 in position 415974: invalid continuation byte
```
The QIF file is from Quicken, not a bank export.  I would expect that Quicken would use the same format since it is their initial format to accept data from banks.
I hope I can get this working as I have been working for a while to get something to read Quicken files.

Cheers, Mel

**Comment by isaacharrisholt** [comment]
Hm, it would seem that your file is not UTF-8 encoded. Could you check the encoding on the for me? I believe there's a Python library called `chardet` that might help

**Comment by ClimberMel** [comment]
Ok, I fixed the file so that it is UTF-8 encoded, but now I get this:
```
File "z:\VSCode-projects\...\qif_parse.py", line 8, in <module>
    qif = Qif.parse('input/test.QIF')
  File "U:\Users\blah\AppData\Local\Programs\Python\Python310\lib\site-packages\quiffen\core\qif.py", line 269, in parse
    AccountType(header_line.split(":")[1]),
  File "U:\Users\blah\AppData\Local\Programs\Python\Python310\lib\enum.py", line 385, in __call__
    return cls.__new__(cls, value)
  File "U:\Users\blah\AppData\Local\Programs\Python\Python310\lib\enum.py", line 710, in __new__
    raise ve_exc
ValueError: 'Bank ' is not a valid AccountType
```
I wonder if a Quicken export qif file is that much different than what you were working with??

**Comment by ClimberMel** [comment]
I was able to import that file into GnuCash as a QIF file if that helps at all...

**Comment by isaacharrisholt** [comment]
Hmm, can you send a copy of the QIF file? With anything personal removed, of course. Also, that one seems like an extraneous space, which we should probably handle, as well as allowing other encodings, so there'll definitely be some changes from this.

**Comment by ClimberMel** [comment]
I'll create a dummy file and post it.  Many banks here in Canada no longer support QIF file format.  I'll check there other formats (besides CSV) to see what they look like.  Quicken now uses QFX which is based on the new standard OFX.  That may be worth exploring if you wish to expand this.

**Comment by isaacharrisholt** [comment]
That would be great. Please let me know once you have the file. I don't have many plans to expand Quiffen to support file types outside of QIF, though if I ever write a parser for OFX, there'll probably be interop of some sort

**Comment by UnusualPi** [comment]
@isaacharrisholt any solve for this yet?  I'm having the same issue.  The QIF file exported from quicken appears to be `ISO-8859-1` and when I convert to `UTF-8`, i get the same `ValueError: 'Bank ' is not a valid AccountType` listed above.

I have quicken R48.15 build 27.1.48.15 running on windows 10.

**Comment by isaacharrisholt** [comment]
> @isaacharrisholt any solve for this yet? I'm having the same issue. The QIF file exported from quicken appears to be `ISO-8859-1` and when I convert to `UTF-8`, i get the same `ValueError: 'Bank ' is not a valid AccountType` listed above.
> 
> I have quicken R48.15 build 27.1.48.15 running on windows 10.

Interesting. I'll open a PR now to expose the encoding so you can override it :)

**Comment by isaacharrisholt** [comment]
Fixed in 2.0.7

**Comment by UnusualPi** [comment]
Thank you @isaacharrisholt !
